documentation/Using-Azure-TestFramework.md

```diff
@@ -76,7 +76,7 @@ To use this option, set the following environment variable before starting Visua
 `tenantId`: The [AAD Tenant ID](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-tenant-id)
 
 
-	TEST_CSM_ORGID_AUTHENTICATION=SubscriptionId={SubId};ServicePrincipal={clientId};ServicePrincipalSecret={clientSecret};AADTenant={tenantId};Environment={env};HttpRecorderMode=Record;
+	TEST_CSM_ORGID_AUTHENTICATION=SubscriptionId={SubId};ServicePrincipal={clientId};Password={clientSecret};AADTenant={tenantId};Environment={env};HttpRecorderMode=Record;
```

Comment:
@akromm this module is a helper module to easily create connection string.
Azure production no longer supports user/password for MS tenant or any 2FA enabled tenant.
So this helper module is created to support majority of the scenarios.
Making this change at the module (powershell module) level will defeat the purpose of this module.
If you want to use "password" key/value pair, you can always create a template connection string using this module and update your connection string with whatever key/value pairs you would like to have in your connection string.
We do not want to change the default behavior of the connection string.